src/main/java/pl/matidominati/GitHubApp/client/GitHubClient.java

```diff
@@ -0,0 +1,19 @@
+package pl.matidominati.GitHubApp.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import pl.matidominati.GitHubApp.client.model.GitHubRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+@FeignClient(value = "gitHub", url = "${github.api.url}")
```

Comment:
github.api.url to jest jakas nasza zmienna konfiguracyjna internalowa -> wewnetrzna. Mozemy zastanowic sie czy nie chcemy jej nazwac np: 
app.github.api.url
I kazda zmienna konfiguracyjna w app properties ktora bedzie zmienna internal to zawierac w sciezce app